helpdesk_mgmt_project/models/project.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models, _
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    ticket_count = fields.Integer(
+        compute='_compute_ticket_count',
+        string="Ticket Count")
```

Comment:
Have you considered maintaining an "open_tickets_count" rather than a total count? It should be interesting IMO, because open tickets are the most urgent ones. If implemeted, if the field is stored, a "project with open tickets" filter should be added to search view.

Reply:
Ok! I get busy!